packages/app-store/routing-forms/components/SingleForm.tsx

```diff
@@ -280,6 +287,7 @@ function SingleForm({ form, appUrl, Page }: SingleFormComponentProps) {
         <FormActionsProvider appUrl={appUrl}>
           <Meta title={form.name} description={form.description || ""} />
           <ShellMain
+            upButton={!!form.description?.length}
```

Comment:
Why is this needed? `form.description` is actually the `subtitle` for Shell. It can handle that itself.